### DISCUSSION TITLE
Google's AlloyDB (SQL) supported?

### DISCUSSION THREAD

**Discussion by ian-santos-moonbridge** [prisma/prisma#13328] [Q&A] [answered]
AlloyDB was announced during Google IO 2022. And its been said that it is based on Postgres.. does this mean it is already supported by prisma?

**Comment by ian-santos-moonbridge** [comment]
https://cloud.google.com/alloydb

**Comment by nurul3101** [comment, marked as answer]
Hey @ian-santos-moonbridge 👋 

If Alloy DB is wire compatible with PostgreSQL you could theoretically start using it with Prisma, but we still need to check what subtle differences are done in alloy, it could be similar to CockroachDB, people were using CockroachDB before we officially created the connector for it, but to fully support it we had to implement changes that were done by Cockroach Labs Team.

**Reply by algoflows** [reply]
Any ideas on how we might set up the connection? Can't find any clear directions in the documentation that help?

**Reply by chrismerkle** [reply]
+1 and following this thread. Any support for AlloyDB @nurul3101? Would be great to see how we can connect to AlloyDB via Prisma data proxy.

**Reply by janpio** [reply]
Please someone open a proper feature request for this, similar to https://github.com/prisma/prisma/issues?q=label%3A%22topic%3A+connector%22+label%3Akind%2Ffeature. We are not tracking discussions for new features, so this will be ignored here.

Thanks.

**Reply by nurul3101** [reply]
Here's a Feature Request for adding support for AlloyDB: #16161 
Please consider adding a 👍 to the feature request so that our product team could prioritise it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
